tools/lexLog_tools.mll

```diff
@@ -118,7 +118,7 @@ let hexanum = "0x" hexa+
 let set = '{' (' '|','|('-'?(num|hexanum)))* '}'
 let alpha = [ 'a'-'z' 'A'-'Z']
 let pteval = '(' ([' ''_'','':''('')']|alpha|digit)+ ')'
-let name = alpha (alpha|digit|'_'| '.')*
+let name = (alpha|'_'|'.') (alpha|digit|'_'| '.')*
 let label = 'L' (alpha|digit)+
```

Comment:
I am a little worried about this but the problem is not this PR and some quick tests seems to work, so we can fix later.